pkg/trace/traceutil/normalize.go

```diff
@@ -297,17 +309,21 @@ func normMetricNameParse(name string) (string, bool) {
 	return string(res), true
 }
 
-func isNormalizedASCIITag(tag string) bool {
+func isNormalizedASCIITag(tag string, checkValidStartChar bool) bool {
```

Comment:
I think this new param name is a bit confusing. Since technically we're still checking that the startChar is a valid character, we just allow for more possible values than before. Any ideas on a better name here? Maybe `allowDigitStartChar`, ideally something better than my example here :P ?

Reply:
I had something similar to this before. I think `allowDigitStartChar` looks good!